doc/ref/spec.html

```diff
@@ -879,6 +882,9 @@ <h3 id="core_syntax">Core Syntax</h3>
           The following binary operators are removed: <code>!=</code> <code>==</code> <code>%</code>
           <code>in</code>
         </li>
+        <li>All function parameters have default expressionss. If no default expression was given
```

Comment:
It took me a while to understand this item. What does it mean to "use an error to force the caller to specify the value"?

Maybe:
"All function parameters have default expressions. If no default expression was given in the original source, a dummy one, which evaluates to an error, is used. "

(Also a typo "expresionss".)

Reply:
Ok so I can't string a sentence together.  But at least the semantics rules are correct? :)